### PR TITLE
feature(hardware-control): add api function for setting the gripper jaw width

### PR DESCRIPTION
**Pull request by ryanthecoder**
# Overview
RLAB-44
RLAB-64

This PR adds an  api method to set a gripper jaw to a specific width
* It adds gripper jaw min and max to the shared data config
* it adds a can message for moving the gripper a specific distance
* it converts the jaw width request to a linear motion request that tells the gripper jaw motor to move a specific number of micrometers

# Changelog

Add a brushed motor linear move request to can messages
connect the can messages up through the ot3-api
add gripper jaw widths to the shared data config

# Review requests

# Risk assessment
low (new feature)

### CODE REVIEW

**Review by sfoster1 — APPROVED** [review]
This looks good to me. Something makes me  feel slightly weird about the way the jaw sizes are defined in config, but we can figure it out as time goes by.

File: api/src/opentrons/hardware_control/ot3api.py [hunk 1]
```diff
@@ -1007,6 +1028,11 @@ async def ungrip(self) -> None:
         await self._ungrip()
         self._gripper_handler.set_jaw_state(GripperJawState.HOMED_READY)
 
+    async def jaw_width(self, jaw_width_um: int) -> None:
```
Comment (ahiuchingau) [inline api/src/opentrons/hardware_control/ot3api.py]: Could we rename `jaw_width` and `_jaw_width` to something that starts with a verb, following the format of the other commands? Like `hold_jaw_width` for example?

Comment (ahiuchingau) [inline api/src/opentrons/hardware_control/ot3api.py]: We should stick with using `mm` here, and only convert it `um` when we construct the CAN message.

**Review by ahiuchingau — CHANGES_REQUESTED** [review]
Logic looks good to me!

I think we should use `mm` in the api and let the hardware take care of distances in `um`.

File: api/src/opentrons/config/gripper_config.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ class GripperConfig:
     pin_one_offset_from_base: Offset
     pin_two_offset_from_base: Offset
     quirks: List[str]
+    jaw_sizes_um: Dict[str, int]
```
Comment (ahiuchingau) [inline api/src/opentrons/config/gripper_config.py]: Nit: could we maybe use mm here?

File: shared-data/gripper/schemas/1.json [hunk 3]
```diff
@@ -90,6 +103,9 @@
       "type": "array",
       "description": "Array of valid quirks for a given gripper",
       "items": { "type": "string" }
+    },
+    "jawSizes_um": {
```
Comment (ahiuchingau) [inline shared-data/gripper/schemas/1.json]: nit: use mm?

File: shared-data/python/opentrons_shared_data/gripper/dev_types.py [hunk 4]
```diff
@@ -117,6 +118,7 @@ def from_dict(cls, data: Dict[str, Any]) -> "GripperDefinitionV1":
                 pin_one_offset_from_base=GripperOffset(**data["pinOneOffsetFromBase"]),
                 pin_two_offset_from_base=GripperOffset(**data["pinTwoOffsetFromBase"]),
                 quirks=data["quirks"],
+                jaw_sizes_um=data["jawSizes_um"],
```
Comment (ahiuchingau) [inline shared-data/python/opentrons_shared_data/gripper/dev_types.py]: same as above

**Review by ahiuchingau — APPROVED** [review]
Looks great to me! 

Just left some nits; feel free to ignore if it actually makes more sense to keep those values in `um` instead!

**Pull request merged**